resources/config.json

```diff
@@ -5,6 +5,10 @@
         "down": 264,
         "up": 265
     },
+    "script": {
+        "file": "bindings/python/testing.py",
```

Comment:
Do we want some kind of demo project that is generated. I hate to say it but a submodule could be useful here, or we might be able to have the CMake toolchain pull down a demo repo?